unreleased.rst

```diff
@@ -35,3 +35,7 @@ HEAD — ongoing
   allowing users to resume a Postgres sandbox ledger without having to again
   specify all packages through the CLI.
   See `#1929 <https://github.com/digital-asset/daml/issues/1929>`__.
+- [DAML Studio] You can now configure the gRPC timeout
+  ``daml.yaml`` via ``scenario-service: {"grpc-timeout": 42}``.
+  This will set the timeout to 42 seconds. This should
```

Comment:
```suggestion
  This will set the timeout to 42 seconds. This option should
```
http://wadler.blogspot.com/2009/03/nonreferential-this.html

Reply:
Thanks! Also changed it to active voice following a suggestion from @bethaitman on a previous PR.